migrations/env.py

```diff
@@ -78,13 +79,17 @@ def process_revision_directives(context, revision, directives):
         poolclass=pool.NullPool,
     )
 
+    # remove the 'render_as_batch' entry from the dictionary so that it's not provided twice
+    configure_args = current_app.extensions['migrate'].configure_args.copy()
+    configure_args.pop('render_as_batch', None)
```

Comment:
hmmmm why is it being provided twice? I think added it recently because it was allegedly not being provided